modules/iap/iap.tf

```diff
@@ -19,17 +19,18 @@ data "google_project" "project" {
 
 # Creates a "Brand", equivalent to the OAuth consent screen on Cloud console
 resource "google_iap_brand" "project_brand" {
-  count             = var.brand == "" ? 1 : 0
+  count             = var.create_brand ? 1 : 0
   support_email     = var.support_email
-  application_title = "${var.app_name}-Application"
+  application_title = "Web Application"
```

Comment:
Why change this?

Reply:
Brand can be only one per project.. Keeping a single application name doesnot make sense.. jo kept generic.. else we can take input for that.